.github/workflows/rituals.yaml

```diff
@@ -81,6 +82,21 @@ jobs:
           pkgs <- installed.packages()[, "Package"]
           sessioninfo::session_info(pkgs, include_base = TRUE)
 
+      - name: Url redirects
+        run: |
+          Rscript -e 'urlchecker::url_update()'
+          # throw an error if man files were updated
+          if [ -n "$(git status --porcelain man)" ]
+          then
+            git status --porcelain
+            >&2 echo "Updated links found in files above"
+            >&2 echo 'Run `urlchecker::url_update()` to fix links locally'
+            exit 1
+          fi
+          # Add locally changed urls
+          git add .
+          git commit -m 'Update links (GitHub Actions)' || echo "No link changes to commit"
```

Comment:
Is there any point to committing the changes?

Reply:
I don't like to commit something that can be automated.  Like documentation updates

Reply:
I'm uneasy about having it change URLs without someone seeing the redirected target first. It's not too implausible that a redirect would end up at the wrong place, say, if some service is temporarily down and has a generic redirect.

Also, am I correct that if roxygen is used, this won't change the original .R files?

Reply:
In the lines above (89-95), I'm checking for any man files were updated. If so, I throw an error as `urlchecker` does not alter `./R` files which would be updated in the roxygen step.

You are correct in that this is open to issues due to services being down.  Maybe we only do this step for `rc-v*` branches?

---------------------------

This goes back to the "outdated code should throw an error" vs "Update all code that can be auto-generated" dilemma. Currently, `rituals.yaml` is in the "throw if it can't be auto-generated" camp.

Reply:
> Maybe we only do this step for rc-v* branches?

I like that idea

Reply:
Implemented in 26c9ac0

Shown that the url step is not run in https://github.com/rstudio/shiny/pull/3249/checks?check_run_id=1691570478